ui/ops/src/routes/devices/components/DeviceSideBarContent.vue

```diff
@@ -20,14 +21,17 @@
     <v-divider v-if="traits['smartcore.traits.OccupancySensor']" class="mt-4 mb-1"/>
     <with-occupancy v-if="traits['smartcore.traits.OccupancySensor']" :name="deviceId" v-slot="{resource}">
       <occupancy-card v-bind="resource"/>
+      <occupancy-history-card v-bind="resource" :name="deviceId"/>
```

Comment:
I'm not sure this needs an entire card on the ui - a menu on the right of the title for the card should be fine.

```
+------------------------------------------+
| Occupancy                          ( ⋮ ) |
|   +------------------------------------+ |
|   | Download History  [ 1d | 1w | 1m ] | |
|   +------------------------------------+ |
|                                          |
+------------------------------------------+
```
Or something like that?